### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by YKerroum**
### Features:

- Add index.html
- Add style.css
- Add test.md
- Create .gitignore
- Set-up Linter
- Modify README.md
- Modify MIT.md

### CODE REVIEW

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @YKerroum 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by YKerroum** [comment]
@omarsalem7 Thank you a lot for your review;
So I have just to use class instead of h1 to pass this step right?

File: README.md [hunk 1]
```diff
@@ -2,71 +2,47 @@
 
 # Project Name
```
Comment (wuletawwonte) [inline README.md:3]: - [ ] [OPTIONAL] You did great using the Microverses template, please also modify its title with your project's title.

**Review by wuletawwonte — APPROVED** [review]
## Hi @YKerroum 👋🏿👋🏿👋🏿, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Your repo is professional 💯 
- Well structured files 💯 
- Pass linter checks 💯
## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (YKerroum) [reply README.md:3]: @wuletawwonte  Thank you so much for your good feedback. I apologize for forgetting to alter it, but I will do so right away.

**Pull request merged**